README.md

```diff
@@ -233,3 +233,5 @@ You can also triage issues which may include reproducing bug reports, or asking
 Or maybe through using pandas you have an idea of your own or are looking for something in the documentation and thinking ‘this can be improved’...you can do something about it!
 
 Feel free to ask questions on the [mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata) or on [Gitter](https://gitter.im/pydata/pandas).
+
+As contributors and maintainers to this project you are expected to abide by pandas' strict code of conduct. More information can be found at: [Contributor Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```

Comment:
```suggestion
As contributors and maintainers to this project, you are expected to abide by pandas' strict code of conduct. More information can be found at: [Contributor Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```

Reply:
Sorry, I should have caught that one as well.

We try our best to follow by an additional code of conduct that is proper, grammatical English 😁

Reply:
@gfyoung  thank you! I totally understand